ext/spl/spl_dllist.h

```diff
@@ -20,6 +20,13 @@
 #include "php.h"
 #include "php_spl.h"
 
+#define SPL_DLLIST_IT_DELETE 0x00000001 /* Delete flag makes the iterator delete the current element on next */
+#define SPL_DLLIST_IT_KEEP   0x00000000
```

Comment:
Reorder it to be in ascending order or values
```suggestion
#define SPL_DLLIST_IT_KEEP   0x00000000
#define SPL_DLLIST_IT_DELETE 0x00000001 /* Delete flag makes the iterator delete the current element on next */
```